crates/grafbase-sdk/src/types/elements.rs

```diff
@@ -0,0 +1,174 @@
+use serde::Deserialize;
+
+use crate::{
+    types::DirectiveSite,
+    wit::{self},
+    SdkError,
+};
+
+/// A list of elements present in the query on which one of the extension's directive was applied on their definition.
```

Comment:
You should elaborate a bit more on this. Maybe even with an example.

Reply:
Like write it like you would have a sock puppet in your hand, explaining this to me.